Tests/LibWeb/Text/input/HTML/Element-checkVisibility.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<style>
+    .display-none {
+        display: none;
+    }
+
+    .content-visibility-parent {
+        content-visibility: hidden;
+    }
+
+    .opacity-hidden {
+        opacity: 0;
+    }
+    .opacity-visible {
+        opacity: 0.1;
+    }
+
+    .visibility-hidden {
+        visibility: hidden;
+    }
+    .visibility-visible {
+        visibility: visible;
+    }
+
+    .content-visibility-auto-hidden {
+        content-visibility: auto;
+    }
+</style>
+
+<div class="display-none"></div>
+
+<div class="content-visibility-parent">
+    <div class="content-visibility-child"></div>
+</div>
+
+<div class="opacity-hidden"></div>
+<div class="opacity-visible"></div>
+
+<div class="visibility-hidden"></div>
+<div class="visibility-visible"></div>
+
+<div class="content-visibility-auto-hidden"></div>
+
+<script src="../include.js"></script>
+<script>
+    asyncTest(done => {
```

Comment:
`asyncTest` is for tests that aren't guaranteed to finish before the page is loaded. We use it a lot for tests that invoke event listener callbacks, for example. It's fine to just use `test` here.

Reply:
Good to know, I've updated it.